packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -139,6 +139,30 @@ class RecoveryPhoneHandler {
 
     return {};
   }
+
+  async exists(request: AuthRequest) {
+    const { uid, emailVerified, mustVerify, tokenVerified } = request.auth
+      .credentials as any;
```

Comment:
Judging by how `credentials` is used, it's a session token, so about a session token type here instead of `any`?